specs/http-client/index.html

```diff
@@ -429,117 +428,116 @@ <h3>Serializing the request content</h3>
 
       <section>
-        <h3>Representing the result content</h3>
-        <p>Instead of being inserted within the <code>http:response</code> element, the content of
-          each body is returned as a single item in the return sequence.  Each item is in the same
-          order (after the <code>http:response</code> element) than the <code>http:body</code>
-          elements.  For each body, the way this item is built from the HTTP response is as
+        <h3>The Response Entity Body</h3>
+        <p>Instead of being inserted within the <a>http:response</a> element, the content of
+          each body is returned as a single item in the returned sequence. Each item is in the same
+          order (after the <a>http:response</a> element) as the <a>http:body</a>
+          elements. For each body, the way this item is built from the HTTP response is as
           follow.</p>
         <p>If the <code>status-only</code> attribute has the value <code>true</code> (default
           is <code>false</code>), the returned sequence will only contain the
-          <code>http:response</code> element (with the headers, but also the empty
-          <code>http:body</code> or <code>http:multipart</code> elements, as if
-          <code>status-only</code> was false), and the following items, representing the bodies
+          <a>http:response</a> element (with the headers, but also the empty
+          <a>http:body</a> or <a>http:multipart</a> elements, as if
+          <code>status-only</code> was <code>false</code>), and the following items, representing the bodies
           content are not generated from the HTTP response.</p>
-        <p>For each body that has to be interpreted, the following rules apply in order to build the
-          corresponding item.  If the body media type is a text media type, the item is a string,
-          containing the body content.  If the media type is an XML media type, the content is
-          parsed and the item is the resulting document node.  If the media type is an HTML type,
+        <p>For each body that has to be parsed, the following rules apply in order to build the
+          corresponding XDM item. If the body media type is a text media type, the item is an <code>xs:string</code>,
+          containing the body content. If the media type is an XML media type, the content is
+          parsed and the item is the resulting XDM <code>document-node()</code>. If the media type is an HTML type,
           the content is <emph>tidied up</emph> and parsed (this process is
-          implementation-dependant) and the item is the resulting document node.  If this is a
-          binary media type, the content is returned as a base64Binary item.  From the previous
-          rules, a result item can then be either a document node (from XML or HTML), a string or a
-          base64Binary.</p>
+          implementation-defined) and the item is the resulting XDM <code>document-node()</code>.  If this is a
```

Comment:
Implementation-dependent is on purpose here, rather than implementation-defined (the process of HTML tidying up HTML might rely on details not necessary to define precisely in order to be conformant.)

Reply:
(and same comment about "document node" as above)

Reply:
Okay I fixed those two.